validator/sawtooth_validator/state/merkle.py

```diff
@@ -169,37 +169,65 @@ def delete(self, address):
 
         return hash_key
 
-    def update(self, set_items, virtual=True):
+    def update(self, set_items, delete_items=None, virtual=True):
         """
 
         Args:
             set_items (dict): dict key, values where keys are addresses
+            delete_items (list): list of addresses addresses
```

Comment:
list of addresses ~~addresses~~

Also, this docstring would be improved by explaining what the addresses are for